test/e2e/testing-manifests/storage-csi/cluster-driver-registrar/README.md

```diff
@@ -0,0 +1 @@
+The original file is (or will be) https://github.com/kubernetes-csi/cluster-driver-registrar/blob/master/deploy/kubernetes/rbac.yaml
```

Comment:
Is it identical to that upstream copy? Then say so :smile:

Reply:
I'm just following the pattern of the other RBAC files in the repo. :P I'd rather not force push yet again for something like this.

Reply:
It's one thing to have out-dated content in the repo. It's another thing to add something that is out-dated from the beginning.

Reply:
Hmm, your file isn't identical to the update file either. Can you please fix that?
```
$ diff -c test/e2e/testing-manifests/storage-csi/cluster-driver-registrar/rbac.yaml /tmp/rbac.yaml 
*** test/e2e/testing-manifests/storage-csi/cluster-driver-registrar/rbac.yaml	2019-02-11 08:32:26.705915735 +0100
--- /tmp/rbac.yaml	2019-02-11 08:33:59.049996292 +0100
***************
*** 16,22 ****
  kind: ClusterRole
  apiVersion: rbac.authorization.k8s.io/v1
  metadata:
!   name: cluster-driver-registrar-runner
  rules:
    - apiGroups: ["csi.storage.k8s.io"]
      resources: ["csidrivers"]
--- 16,22 ----
  kind: ClusterRole
  apiVersion: rbac.authorization.k8s.io/v1
  metadata:
!   name: cluster-driver-registrar-role
  rules:
    - apiGroups: ["csi.storage.k8s.io"]
      resources: ["csidrivers"]
***************
*** 26,32 ****
  kind: ClusterRoleBinding
  apiVersion: rbac.authorization.k8s.io/v1
  metadata:
!   name: csi-cluster-driver-registrar-role
  subjects:
    - kind: ServiceAccount
      name: csi-cluster-driver-registrar
--- 26,32 ----
  kind: ClusterRoleBinding
  apiVersion: rbac.authorization.k8s.io/v1
  metadata:
!   name: csi-driver-registrar-role
  subjects:
    - kind: ServiceAccount
      name: csi-cluster-driver-registrar
***************
*** 34,38 ****
      namespace: default
  roleRef:
    kind: ClusterRole
!   name: cluster-driver-registrar-runner
    apiGroup: rbac.authorization.k8s.io
--- 34,38 ----
      namespace: default
  roleRef:
    kind: ClusterRole
!   name: cluster-driver-registrar-role
    apiGroup: rbac.authorization.k8s.io
```

Reply:
I would like to merge this as-is and update it once the cluster-driver-registrar repo is updated with this RBAC file.

Reply:
I'll say, there's nothing outdated about the sentence as-is. At any given point, the link will point to either a copy of the YAML file or will eventually be updated to be the same as the YAML file.

Reply:
Why should the cluster-driver-registrar be updated with this RBAC file here? The source of truth is the cluster-driver-registrar repo, not the E2E manifest directory.

Regarding "original file is (or will be)": I used the "(or will be)" part when I added those other README files because at that time, the RBAC files had not been moved to separate repos yet, so I had to be vague. Now the file exists and that reason for being vague no longer holds. My opinion is that the other READMEs should be updated and no new READMEs with this wording should be added. Just my 2 cents. Feel free to ignore it.